components/Button/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
```

Comment:
Is this the structure Sean employs? I'm not familiar with styled-comments, is it common to separate styles into a styles.js file?

Reply:
I know he's a big fan of styled components and he demo'ed them for me in that particular component. However, I haven't been able to really figure out how to use them otherwise. How do you do the styling? In short, I'm not really sure because all the structure is pretty new to me.

Reply:
Styled components are ok with me, I'm not familiar with them but I would like to become more familiar as I like that idea of styling, I was just curious if he stored styles.js in the component folder. It sounds like he does so I'm going to close the comment and we can leave it as is, I usually store my CSS/SCSS files in the same folder as my component so it's an idea I'm familiar with.